baseplate/lib/metrics.py

```diff
@@ -501,7 +505,13 @@ def metrics_client_from_config(raw_config: config.RawConfig) -> Client:
     """
     cfg = config.parse_config(
         raw_config,
-        {"metrics": {"namespace": config.String, "endpoint": config.Optional(config.Endpoint)}},
+        {
+            "metrics": {
+                "namespace": config.String,
+                "endpoint": config.Optional(config.Endpoint),
+                "histogram_sampling_rate": config.Optional(config.Float),
```

Comment:
Is it better to pull the `histogram_sampling_rate` out of the config here in the same place where we get a bunch of other config data, or later on where we wouldn't have to pass it through so many constructors?